src/gpu/intel/CMakeLists.txt

```diff
@@ -21,13 +21,16 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "IntelLLVM" OR CMAKE_CXX_COMPILER_ID MATCHES "
 endif()
 
 file(GLOB SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/*.[ch]pp)
-set(DIRS "bnorm;gemm;reduction;rnn")
+file(GLOB SOURCES_GEMM ${CMAKE_CURRENT_SOURCE_DIR}/gemm/*.[ch]pp)
+list(APPEND SOURCES "${SOURCES_GEMM}")
+set(DIRS "binary;bnorm;conv;concat;eltwise;gnorm;ip;lnorm;lrn;matmul;pooling;prelu;reduction;reorder;resampling;rnn;sdpa;shuffle;softmax;sum;zeropad")
```

Comment:
Since this list is abbreviation-heavy, `pooling` -> `pool` would fit better and be better aligned with the benchdnn driver name.